src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="form"
```

Comment:
Please check formatting for each case
![image](https://user-images.githubusercontent.com/54936044/231732375-130acebd-654c-463c-94e9-2cdf46ed6abd.png)